sdk/src/main/scala/com/horizen/account/state/GasPool.scala

```diff
@@ -12,6 +12,8 @@ class GasPool(initialGas: BigInteger) extends SparkzLogging {
 
   def getUsedGas: BigInteger = initialGas.subtract(currentGas)
 
+  def getInitialGas: BigInteger = initialGas
```

Comment:
method is not used.
In case we want to have it as a part of the interface we may do:
`class GasPool(initialGas: BigInteger)` -> `case class GasPool(initialGas: BigInteger)`

Reply:
This class has mutable internal state, aren't case classes supposed to be immutable? :thinking:

Reply:
good point, then we may have:
 class GasPool(**val** initialGas: BigInteger)

Reply:
done